tools/bin/run-harness-process.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   containers:
   - name: main
     image: airbyte/$HARNESS:dev
-    args: ["$CONNECTOR_IMAGE_NAME", "$DATASET"]
+    args: ["$CONNECTOR_IMAGE_NAME", "$DATASET", "$STREAM_NUMBER"]
```

Comment:
Make sure that this doesn't break source-harness as it still expects either 1 or 2 arguments but not 3.
We might need to create a separate run-harness-process yamls

Reply:
This won't because it expects a different number of args and will throw an error

https://github.com/airbytehq/airbyte/blob/381f49e250d77e66974aa0d28400dcceb1566b34/airbyte-integrations/connectors-performance/source-harness/src/main/java/io/airbyte/integrations/source_performance/Main.java#L35-L38